src/core/components/AMInstallButton/index.js

```diff
@@ -321,13 +321,17 @@ export class AMInstallButtonBase extends React.Component<InternalProps> {
         {this.showLoadingAnimation() ? (
           <CSSTransition key="loading" {...transitionProps}>
             <div
-              className={makeClassName('AMInstallButton-loading', {
-                'AMInstallButton-loading--puffy': this.props.puffy,
+              className={makeClassName('AMInstallButton-loading-button', {
+                'AMInstallButton-loading-button--puffy': this.props.puffy,
               })}
+              title={this.getButtonText()}
             >
-              <span className="AMInstallButton-loading-icon">
-                <Icon alt={this.getButtonText()} name="loading" />
-              </span>
+              <div className="AMInstallButton-loader">
+                <div className="AMInstallButton-loader-container">
+                  <div className="AMInstallButton-loader-ball" />
+                </div>
+                <span className="visually-hidden">{this.getButtonText()}</span>
```

Comment:
can the `this.getButtonText()` be set to a var since it's used a few times?

Reply:
yep